website_event_sale_joined_constraint/static/src/js/website_event_ticket_details.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Camptocamp SA (https://www.camptocamp.com).
+// License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+odoo.define("website_event_sale_joined_constraint.ticket_details", function (require) {
+    "use strict";
+
+    const ticketDetailsWidget = require("website_event.ticket_details");
+
+    ticketDetailsWidget.include({
+        // --------------------------------------------------------------------------
+        // Private
+        // --------------------------------------------------------------------------
+
+        /**
+         * @private
+         * @returns {integer} Number of selected parent tickets
+         */
+        _getParentTicketCount: function () {
+            var ticketCount = 0;
+            this.$(".parent_ticket_select").each(function () {
+                ticketCount += parseInt($(this).val(), 10);
+            });
+            return ticketCount;
+        },
+
+        /**
+         * @private
+         * @returns {integer} Number of selected child tickets
+         */
+        _getChildTicketCount: function () {
+            var ticketCount = 0;
+            this.$(".child_ticket_select").each(function () {
+                ticketCount += parseInt($(this).val(), 10);
+            });
+            return ticketCount;
+        },
+
+        // --------------------------------------------------------------------------
+        // Handlers
+        // --------------------------------------------------------------------------
+        /**
+         * @override
+         */
+        _onTicketQuantityChange: function () {
```

Comment:
if it's an override why don't you call `this._super.apply(this, arguments);`?

Reply:
Because I forgot :wink: 
This has no impact but could have one if there were another override elsewhere, I'll update my PR